web/src/app/converters/submission-data-converter.ts

```diff
@@ -65,27 +61,28 @@ function taskDataPbToModel(pb: Pb.ITaskData[], job: Job): SubmissionData {
       takePhotoResult,
     } = taskData;
 
+    let value = null;
```

Comment:
@rfontanarosa Also, can you please make the treatment of skipped explicit, and log the invalid state where `value` is non-null but `skipped` is `true`? Something like:

```ts
if (skipped) {
  if (value) {
    console.warn(`Ignoring unexpected value ${value} on skipped task`);
  }
  return new Result(null, true);
}
```